crates/bevy_render/src/render_graph/node.rs

```diff
@@ -336,7 +336,7 @@ impl Node for RunGraphOnViewNode {
 
 /// This trait should be used instead of the [`Node`] trait when making a render node that runs on a view.
 ///
-/// It is intended to be used with [`ViewNodeRunner`]
+/// It is intended to be used with [`ViewNodeRunner`]'
```

Comment:
```suggestion
/// It is intended to be used with [`ViewNodeRunner`]
```

I think that single quote was a typo :)